addons/goldpinger/3.6.1-5.4.3/install.sh

```diff
@@ -0,0 +1,34 @@
+
+function goldpinger() {
+    local src="$DIR/addons/goldpinger/3.6.1-5.4.3"
+    local dst="$DIR/kustomize/goldpinger"
+
+    cp "$src/kustomization.yaml" "$dst/"
+    cp "$src/goldpinger.yaml" "$dst/"
+    cp "$src/servicemonitor.yaml" "$dst/"
+
+    if [ -n "${PROMETHEUS_VERSION}" ]; then
+        insert_resources "$dst/kustomization.yaml" servicemonitor.yaml
+    fi
+
```

Comment:
This monitor should be packaged with the prometheus add-on

Reply:
Perhaps we should add an addon_post_init hook to work around this dependency on the order the add-ons are applied?